frontend/dashboard/components/OrgRepoList/OrgReposList.tsx

```diff
@@ -43,15 +50,15 @@ export const OrgReposList = ({ user, organizations }: OrgReposListProps) => {
         {getReposLabel({ selectedContext, orgs: organizations, t })}
       </Heading>
       <RepoList
-        repos={reposWithStarred.filter((repo) => !repo.name.endsWith('-dataModels'))}
-        isLoading={isLoadingSearchResults || areStarredReposPending}
-        onPageSizeChange={setPageSize}
+        repos={reposWithStarred.filter((repo) => !repo.name.endsWith('-datamodels'))}
```

Comment:
Same in this file with the naming to make it more precise to not being confused with reposWithStars. 

And if we do as proposed above we do not need this filter either, I guess?

Reply:
There is no const named `reposWithStars`?